build.sbt

```diff
@@ -1,31 +1,52 @@
 import Dependencies._
 import org.scalafmt.sbt.ScalafmtPlugin.autoImport.scalafmtOnCompile
 
-ThisBuild / scalaVersion := "2.12.13"
+lazy val scala212 = "2.12.13"
+lazy val scala213 = "2.13.5"
+lazy val supportedScalaVersions = List(scala212, scala213)
+
+
+ThisBuild / scalaVersion := scala212
 ThisBuild / version := "0.1.0-SNAPSHOT"
 ThisBuild / organization := "org.virtuslab"
 ThisBuild / organizationName := "VirtusLab"
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
+lazy val commonSettings = Seq(
+  scalafmtOnCompile := true,
+  scalacOptions ++= Seq(
+    "-deprecation",
+    "-encoding",
+    "UTF-8",
+    "-feature",
+    "-language:_",
+    "-Xfatal-warnings",
+    "-Xlog-reflective-calls",
+    "-Xlint:_",
+    "-Ybackend-parallelism",
+    "8",
+    "-Ywarn-dead-code",
+    "-Ywarn-unused:-imports,_",
+    "-unchecked"),
+  libraryDependencies ++= deps
+)
+
+
 lazy val root = (project in file("."))
```

Comment:
Nit: does project-matrix allow of getting rid of this top-level boilerplate module somehow?

Reply:
According to the example on the plugins Github page it does. I decided to preserve it in case it turns out to be needed again.

Reply:
Oh ok let's leave then